saleor/graphql/order/mutations/orders.py

```diff
@@ -580,16 +583,35 @@ def clean_input(cls, info, instance, data):
             raise ValidationError(validation_errors)
         return data["input"]
 
+    @classmethod
+    def clean_instance(cls, info, instance):
+        if instance.status == OrderStatus.DRAFT:
+            raise ValidationError(
+                {
+                    "orderId": ValidationError(
+                        "Provided order cannot be draft.",
+                        code=InvoiceErrorCode.NOT_READY,
```

Comment:
Do we `NOT_READY` in some other cases as well? I'm wondering if it is clear here. Maybe something more explicit like `INVALID_STATUS` would be more appropriate?

Reply:
We use it also for invoice when trying to send it and due to some async stuff it is not marked as ready yet. But maybe it's good to use new status here, furthermore now i'm using InvoiceErrorCode on field orderId... So it would be for the best imo to create `INVALID_STATUS` on `OrderErrorCode` and use it here.

Reply:
It should be added to InvoiceErrorCode so its easier for fronted what error codes can happen in invoice-related mutations.

Reply:
Done